source/pug/templates/master.pug

```diff
@@ -78,8 +78,8 @@ html
               a(href="https://www.facebook.com/mozilla" class="footer-link-facebook") Facebook
             li.col-auto
               a(href="https://creativecommons.org/licenses/by/4.0" class="footer-link-cc-license") License
-            //- li.col-auto
-            //-   a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            - li.col-auto
+            -   a(href="https://www.mozilla.org/en-US/about/governance/policies/participation/" class="footer-link-participation-guidelines") Participation Guidelines
```

Comment:
remove the `/en-US` part of the URL.

Reply:
also, let's leave the class name as what it was before.

When we land your other PR, we will need to update the version of mofo-bootstrap here so we can change the class name here in that patch.

Reply:
ok